models/LogLevel.js

```diff
@@ -1,11 +1,10 @@
 module.exports = require('./Enum').create({
   values: [
-    'FATAL',
-    'ERROR',
-    'WARN',
-    'INFO',
-    'DEBUG',
-    'TRACE'
-  ],
-  autoLowerCase: true
```

Comment:
This didn't seem to do anything and/or wasn't being used. Maybe @austinkelleher can comment on the intention there?